NEWS.md

```diff
@@ -47,6 +68,9 @@ The feed template for the URLHaus feed contained a spelling error:
 The correct name for the parameter "delimeter" is "delimiter". Please fix your configured bots.
 The `intelmqctl upgrade-config` command automatically fixes a configuration if the misspelling is detected.
 
+#### IntelMQ-Manager -> IntelMQ-API
+`IntelMQ-Manager` is getting replaced by `IntelMQ-API`. The `IntelMQ-Manager` release only contains the files for web frontend. Install the `IntelMQ-API`, in order to upgrade your local installation. Follow the instructions in our [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html) to install `IntelMQ-API` & setup authentication. Unfortunately you can't keep current authentication users, you have to add users as described in the [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html#adding-a-user).
```

Comment:
I wouldn't say that the Manager was replaced by the API, but the Manager's PHP-based backend was replaced by the newly developed Python-based IntelMQ API.

Please also add, that those who use the deb/rpm packages automatically receive the IntelMQ API via package dependencies upon upgrade.